packages/core/src/writer.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ServerResponse as Response} from 'http';
+
+// tslint:disable:no-any;
+export type OperationRetval = any;
+// tslint:enable:no-any
+
+export function writer(
```

Comment:
'writer' was the function name Ritchie used in previous design call. I can change writer --> writeResultToResponse